cmd/kubeadm/app/cmd/phases/upgrade/node/kubeletconfig.go

```diff
@@ -78,9 +78,13 @@ func runKubeletConfigPhase() func(c workflow.RunData) error {
 		}
 		src := filepath.Join(kubeletDir, constants.KubeletConfigurationFileName)
 		dest := filepath.Join(backupDir, constants.KubeletConfigurationFileName)
-		fmt.Printf("[upgrade] backing up kubelet config file to %s\n", dest)
-		if err := os.Rename(src, dest); err != nil {
-			return errors.Wrap(err, "error backing up the kubelet config file")
+		if !dryRun {
+			fmt.Printf("[upgrade] backing up kubelet config file to %s\n", dest)
+			if err := os.Rename(src, dest); err != nil {
+				return errors.Wrap(err, "error backing up the kubelet config file")
+			}
+		} else {
+			fmt.Printf("[upgrade] Would back up kubelet config file to %s\n", dest)
```

Comment:
```suggestion
			fmt.Printf("[dryrun] Would back up kubelet config file to %s\n", dest)
```
i think we should have [dryrun] for consistency.
q: do we have words after the ] starting with uppercase (w vs W)? might be a good idea to correct these in ghe new backup code printfs too.

Reply:
It seems all `fmt.Printf("[upgrade] ` are starting with uppercase.

Reply:
ik

Reply:
![image](https://user-images.githubusercontent.com/2010320/208067694-080712d8-21c1-46e8-a62f-f387d3c9b180.png)

Updated.
But, we should keep it consistent to start with uppercase.